helm-chart/renku/templates/ui/ui-client-deployment-template.yaml

```diff
@@ -25,14 +25,17 @@ spec:
         app: ui
         release: {{ .Release.Name }}
     spec:
-      {{- if and .Values.ui.client.privacy.enabled .Values.ui.client.privacy.page.enabled }}
+      {{- if .Values.ui.client.privacy.page.enabled }}
       volumes:
         - name: privacy
           configMap:
             name: {{ .Values.ui.client.privacy.page.configMapName | default (printf "%s-privacy-sample" (include "renku.fullname" .)) | quote }}
```

Comment:
This does not match with the config map! It uses `{{ template "ui.fullname" . }}-privacy-sample` for the default one.

Reply:
Yes, I fixed that this morning.